### DISCUSSION TITLE
Use freeruns with tearaway pants to generate adventures?

### DISCUSSION THREAD

**Discussion by KnivesIV** [loathers/garbage-collector#2048] [Ideas]
Title. Not sure if this is ever actually worth it without banderboots though?

**Comment by Tapionator** [comment]
You'd have to find a zone that is exclusively plant based enemies. Maybe at the Fun Guy Mansion at Spring Break?

**Comment by Tapionator** [comment]
It is def worth it for players who used soups on their banders and have a ton of left over free runs at the very least. Unsure how many free runs loopers have remaining at the end of the run after getting shadow rift buffs and/or Gingerbread candies.

**Comment by Tapionator** [comment]
I just threw the idea in a Feature request.
